graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/user/GqlPrincipal.java

```diff
@@ -46,7 +47,9 @@ static DXPaginatedData<GqlGroup> getGroupMembership(String localPath,
 
         List<String> paths = groupManagerService.getMembershipByPath(localPath);
         Stream<GqlGroup> stream = paths.stream()
-                .map(path -> groupManagerService.lookupGroupByPath(path).getJahiaGroup())
+                .map(groupManagerService::lookupGroupByPath)
```

Comment:
`lookupGroupByPath()` can return `null` (if the group does not exist anymore).
In the new test (`site.cy.ts`), a new site is created which also creates a new group for it.
And because  `groupManagerService.getMembershipByPath(localPath)` gets the list from a cache (which for some reasons is not flushed when the site is deleted), it's returning a group that no longer exists.
Which was causing an existing test to fail (`user.cy.ts`)